exe/slow_query_exporter

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env ruby
```

Comment:
Interesting. I could've sworn it was `bin` by default at some stage. I wonder why they changed it?